shell/platform/android/io/flutter/plugin/platform/VirtualDisplayController.java

```diff
@@ -128,6 +129,15 @@ public int getRenderTargetHeight() {
   }
 
   public void resize(final int width, final int height, final Runnable onNewSizeFrameAvailable) {
+    // When 'hot reload', although the resize method is triggered, the size of the native View has
+    // not changed.
+    if (width == getRenderTargetWidth() && height == getRenderTargetHeight()) {
+      return;
```

Comment:
After reading the calling code, we do need to run the onNewSizeFrameAvailable in this case.